### DISCUSSION TITLE
Clearer Text and Explanations Of OpenMage 1 and 2 on Project Website

### DISCUSSION THREAD

**Discussion by Dirtyhair1** [OpenMage/magento-lts#1732] [Questions & Answers]
Hey guys,

I hope all is well!

First time posting here so be nice :)

I feel like there are a lot of store owners out there who don't understand the difference between OpenMage 1 and OpenMage 2, and infact, some don't even know there is an OpenMage 2.

I am happy to write something up and explain it clearly and maybe it can be posted on the project website? For example, I was in touch with a fairly large merchant the other day, they have looked at OpenMage but they are concerned about ongoing integration with new technology and things like that. I replied with the example that PHP8 support is being worked on in OpenMage and it is even ahead of Magento2 in this area (I think.)

Anyway, main idea is to simply make it clearer and easier for people to understand. Many of you have multiple clients who you have migrated over to OpenMage, but there are also a lot of people who self manage or who use agencies that don't follow the project that closely.

Anyway, let me know what you think. 

I think the process would be:

- explain what OpenMage 1 and 2 are

- get a list of "projects" or something that are being worked on in OpenMage 2

- keep updating that list of projects so people can see what's happening and how it's going.

Thoughts?

Thanks!

**Comment by pauldpauld** [comment]
Speaking as a "self-managed", I would really like to see this.

I have been using Magento since 1.4.1.1. and am currently on 1.7, I am looking to upgrade to OpenMage and would appreciate knowing the differences between Openmage 1.9 and Openmage 20
Pehaps also, this could cover (just generally) how much we could bring over (in database terms) between old Magento and new Openmage (1.9 and 20).

Thanks, Paul

**Reply by Dirtyhair1** [reply]
Hi Paul, yep that is what I am talking about. 

Can you explain what confuses you, or what the roadblocks are? When you first heard about OpenMage or looked at the website, what did you initially think?

Just trying to get some background idea to hopefully cover those points. 

Thanks!

**Reply by pauldpauld** [reply]
Thanks for taking the time with this, in no particular order; (and writing my answer, I am understanding how involved your question is!)

- If I upgrade to OM 1.9 I would like to see; how I ACTUALLY get the new OM 1.9 onto my server (I run my own) what steps I need to take and what extra packages I might need to install. Comments on what has changed between Magento 1.9 and OM 1.9 (just generally, not variable by variable !!)

- If I upgrade to OM 20
same remarks as 1.9 but also covering what has changed in general, (in other words, what might bite me when I have OM20 installed with all my customer details, history, products. etc).

- Looking at roadblocks/confusion I think a page similar to; "hey, you don't want to read pages and pages, look, here are the steps to take, one after the other, if step 5 does this, then look at this comment". 
BUT when things don't install, perhaps a more in-depth collection of steps?

- How do I get either package to auto install new patches? the team here are upgrading all the time it seems (I think it's brilliant they do that!) that it would make sense to use Cron keep my version up to date on a daily/weekly basis. AND thinking about it, a way to roll-back the changes if something fails.

- I have avoided Composer in the past, as I have downloaded and installed from files. IF it seems that it is the only real way to continue, perhaps a brief "install composer like this", or a link to a step by step.
- There seem to be a range of extensions on the OM site, do they all work on OM 1.9 and OM20? I understand that there is also a huge collection of untested/unchecked extensions also available. For someone like me, if I am told an extension works with OM1.9 for example, and it doesn't work for me, I know I've done something wrong and I will re-check until I get it working. If it is an untested extension I will just uninstall it and move on! 

I hope my ramblings are of some help to you? 
Just to expand slightly, I DO read all the postings from OM, and I find them helpful  (not that I understand them all), but after I have read a few in-depth posts about about something, I find myself just a little lost, and then thinking about a finding a cup of coffee :-)

Thanks again!
-

**Comment by kiatng** [comment]
I forgot that there is an official [OpenMage](https://www.openmage.org/) site, which I hardly visit. As pointed by @Flyingmana in discord, its repo is [here](https://github.com/OpenMage/OpenMage.github.io). Check it out to see what contents need to be updated/expanded on. I agree it's not clear which branch to install for new users as there is not much info/guide.

**Reply by Dirtyhair1** [reply]
Hi @kiatng ok great thanks for that. I am not a developer (at all) but I will try to write/make a clear explanation. Would you possible be ok to assist in posting it? (or the process to through.) Cheers :)

**Reply by kiatng** [reply]
Sure, even though I am not sure how to post it, but I'll try to figure it out and assist you.
